### PR TITLE
cleanups in dependencies/ dead files / npm scripts

### PR DESCRIPTION
**Pull request by eugene-matvejev**

### CODE REVIEW

**Comment by andgreendigital** [comment]
Can we update this PR? 🙂

File: package.json [hunk 1]
```diff
@@ -59,21 +59,15 @@
...
-    "setup": "node ./bin/setup.js",
     "start": "gatsby develop",
     "serve": "gatsby serve",
     "build": "gatsby build",
     "clean": "gatsby clean",
     "deploy": "gatsby build --prefix-paths && gh-pages -d public",
     "test": "jest",
-    "cleanup-repository": "rimraf bin contentful && yarn remove contentful-import chalk inquirer rimraf ",
-    "lint": "eslint --ext .jsx --ext .js .",
-    "lint-fix": "eslint --ext .jsx --ext .js --fix ."
```
Comment (andgreendigital) [inline package.json]: Can we keep `lint-fix` since it isn't linked to the hooks anymore it won't affect anyone but I'd like to use it! 🙂

Comment (eugene-matvejev) [reply package.json]: easy

**Comment by eugene-matvejev** [comment]
there is a pattern to name commands <namespace>:<command>:<subcommand> etc

**Pull request merged**